docs/make.jl

```diff
@@ -1,7 +1,12 @@
+module BuildDocu
 using Oscar
 local_build = false
 bla = normpath(joinpath(dirname(pathof(Oscar)), "..", "docs", "make_work.jl"))
 include(bla)
```

Comment:
While we are at it, could also improve code readability:
```suggestion
include(normpath(joinpath(Oscar.oscardir, "docs", "make_work.jl")))
```